index.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>My Funny Friend and Me</title>
+    <script type='text/javascript' src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <script type='text/javascript' src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.js"></script>
+    <script type="text/javascript" src='//cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/js/toastr.min.js'></script>
```

Comment:
Nice job setting all of these external resources. Notice on line 8 that the `src=` starts directly with `//`. When you deploy, if you use this syntax, it allows the resource to determine what protocol is necessary. In the case of working locally, for testing, you'll need to be explicit about the protocol otherwise it'll assume you want the `FILE://` protocol.